sklearn/gaussian_process/_gpr.py

```diff
@@ -48,16 +48,17 @@ class GaussianProcessRegressor(MultiOutputMixin,
         passed, the kernel "1.0 * RBF(1.0)" is used as default. Note that
         the kernel's hyperparameters are optimized during fitting.
 
-    alpha : float or array-like of shape (n_samples), default=1e-10
```

Comment:
Sorry for being late to the party. I am fine with the changes, but why is the type documentation changed here? since an array-like input for alpha is still possible, it should be mentioned here. Or will there be another PR removing this feature?
Just my two cents.

Reply:
Sorry I thought I had changed that back to how it was; will do so now.